RevenueCatUI/CustomerCenter/URLUtilities.swift

```diff
@@ -74,6 +74,14 @@ enum URLUtilities {
 
 }
 
+extension URL {
+
+    var isDeeplink: Bool {
+        !(scheme?.hasPrefix("http") ?? false)
```

Comment:
I wonder if we should match exactly `http` and `https`... Unlikely that someone would create a deep link schema like `httpmyschema` but 🤷

Reply:
not that crazy for someone doing that. If it's an app that starts with http it could happen